openerp/addons/base/migrations/9.0.1.3/pre-migration.py

```diff
@@ -59,6 +59,7 @@ def cleanup_modules(cr):
         cr, [
             ('contacts', 'mail'),
             ('marketing_crm', 'crm'),
+            ('email_template', 'mail'),  # mail_template class
```

Comment:
This one is already done in https://github.com/OCA/OpenUpgrade/blob/9.0/addons/mail/migrations/9.0.1.0/pre-migration.py#L25. Maybe it's better to move it here, but remove it from the other place then.